PicApp_git/PicApp/database/migrations/2020_07_30_133530_create_user_follow_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUserFollowTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_follow', function (Blueprint $table) {
+            $table->id();
+            $table->biginteger('user_id')->unsigned()->index();
+            $table->biginteger('follow_id')->unsigned()->index();
```

Comment:
followテーブルのidに見えてしまうので `follow_user_id` のほうが良さそうです
```suggestion
            $table->biginteger('follow_user_id')->unsigned()->index();
```